hangupsbot/plugins/_chatbridge/chatbridge_syncrooms.py

```diff
@@ -59,25 +59,55 @@ def _send_to_external_chat(self, config, event):
         message = event.passthru["original_request"]["message"]
         image_id = event.passthru["original_request"]["image_id"]
 
+        is_action = event.passthru["chatbridge"].get("source_action")
+
         if not message:
             message = ""
 
+        if any(a.type == 4 for a in event.conv_event._event.chat_message.annotation):
```

Comment:
Good catch, hadn't considered the `FakeEvent` case -- should hopefully be covered now.